### ISSUE TITLE
Installation of Management Console Fails

### ISSUE THREAD

**Issue by derickrn** [deepfence/ThreatMapper#41]
Provided the docer-compose.yml file, when executing the following command:

```docker-compose -f docker-compose.yml up -d```

I am unable to browse to the management console as not all containers are started, specifically the deepfence-es-master container which presents the following errors:
```
Exception in thread "main" java.lang.RuntimeException: starting java failed with [137]
output:

error:
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
	at org.elasticsearch.tools.launchers.JvmErgonomics.flagsFinal(JvmErgonomics.java:123)
	at org.elasticsearch.tools.launchers.JvmErgonomics.finalJvmOptions(JvmErgonomics.java:88)
	at org.elasticsearch.tools.launchers.JvmErgonomics.choose(JvmErgonomics.java:59)
	at org.elasticsearch.tools.launchers.JvmOptionsParser.main(JvmOptionsParser.java:95)
```

Would appreciate any guidance.

**Comment by shyam-dev** [comment]
This is most likely an elasticsearch setting that needs to be done.

Can you set the following value in your system - https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html

Once that is done, please do docker-compose -f docker-compose.yml down, followed by docker-compose -f docker-compose.yml up -d

Thanks

**Comment by derickrn** [comment]
Running this on windows I had to do the following based on what you linked for anyone else who runs into this same problem:
```wsl -d docker-desktop sysctl -w vm.max_map_count=262144```

From that I'm no longer getting any errors but it seems that I still cannot access the management console when browsing to any interface address on my host. I have disabled the Windows Firewall as a measure as well but still nothing.

**Comment by sandman137** [comment]
Appears that you are installing deepfence console on WSL, correct? WSL as a platform for console is officially not supported but should work theoretically and it appears that you might have gotten it up and running already.

While we try this internally, could you please share the console logs? You could also join us on community slack for sharing log files etc.

**Comment by ramanan-ravi** [comment]
Hello @derickrn,

Please join community slack here: https://join.slack.com/t/deepfence-community/shared_invite/zt-iicjmffl-8G2hY74AyKv9v4xHwfJ~3g

**Comment by shyam-dev** [comment]
Hello @derickrn,

Please extract and run the attached shell script in your system. It will save the logs under /tmp/deepfence_logs.tar.gz. Please post that file here and we will take a look at it.


[get-logs.sh.gz](https://github.com/deepfence/ThreatMapper/files/5466060/get-logs.sh.gz)

**Comment by sandman137** [comment]
This is no longer relevant, closing it.
